src/jit/morph.cpp

```diff
@@ -18416,6 +18417,29 @@ class LocalAddressVisitor final : public GenTreeVisitor<LocalAddressVisitor>
         m_compiler->fgMorphLocalField(node, user);
         INDEBUG(m_stmtModified |= node->OperIs(GT_LCL_VAR);)
     }
+
+    //------------------------------------------------------------------------
+    // UpdateImplicitByRefCounter: updates the ref count for implicit byref params.
```

Comment:
Maybe spell this out a bit more? `UpdateEarlyRefCountForImplicitByRef` ?

Reply:
Thanks, fixed.